### PR TITLE
Few improvements

### PR DESCRIPTION
**Pull request by SilentException**
Thanks for the cool script. 
I've made a couple of changes so here is a PR. It's not much but maybe you'll consider merging this.

shortix.sh:
- remember the last update and only run if compatdata has newer directories
- don't break with read when protontricks could not be found (service?)
- check for existence of compatdata

shortx_install.sh
- fixed systems ->systemd typo
- ask if the service should be set up...
- wait for input after install/update

### CODE REVIEW

File: shortix.sh [hunk 1]
```diff
@@ -18,7 +15,7 @@ shortix_script () {
     elif [ "$(command -v $PROTONTRICKS_FLAT)" ]; then
         PROTONTRICKS=$PROTONTRICKS_FLAT
     else
-        read -p "Protontricks could not be found! Please install it. Press ENTER to exit."
```
Comment (Jannomag) [inline shortix.sh:21]: I noticed that sometimes the terminal window closes, if the script gets executed manually. This was my attempt to provide a bit of error information if someone is wondering why nothing happened - this won't work if it's run as a service of course

File: shortix.sh [hunk 2]
```diff
@@ -42,13 +39,31 @@ shortix_script () {
...
 
 if [ ! -f $FIRSTRUN ]; then
     shortix_script
-    touch $FIRSTRUN
-elif [ $(find $COMPDATA -mmin -$TIME -type d) ]; then
-    shortix_script
+    touch "$FIRSTRUN"
+else
+    dorun=1
+    # if there is lastrun file, only run if there are compatdata folders newer than the lastrun file timestamp
```
Comment (Jannomag) [inline shortix.sh:56]: That makes completely, sense, thanks for this!

**Review by Jannomag — APPROVED** [review]
Thanks for this PR, this are good improvements!

File: shortix.sh (hunk 1 above)
Comment (SilentException) [reply shortix.sh:21]: I am executing the script from shell/terminal so the window doesn't actually close.
It is not that common to have "blocking" messages in scripts. But I understand. Perhaps the middle ground would be to replace the two error messages and done message with read -p ... -t 10 argument - the read timeout in seconds.

**Pull request merged**